platforms/blabsy/src/components/tweet/tweet-actions.tsx

```
@@ -28,250 +28,277 @@ import type { Tweet } from '@lib/types/tweet';
+            );
+        };
 
-  const userIsFollowed = following.includes(createdBy);
+    const userIsFollowed = following.includes(createdBy);
 
-  const currentPinModalData = useMemo(
-    () => pinModalData[+tweetIsPinned],
-    // eslint-disable-next-line react-hooks/exhaustive-deps
-    [pinOpen]
-  );
+    const currentPinModalData = useMemo(
```

Biome (1.9.4)

[error] This hook does not specify all of its dependencies: tweetIsPinned

This dependency is not specified in the hook dependency list.


(lint/correctness/useExhaustiveDependencies)